pom.xml

```diff
@@ -121,12 +121,12 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <hk2-bridge.version>2.6.0</hk2-bridge.version>
-        <jersey.version>2.29</jersey.version>
-        <jetty.version>9.4.28.v20200408</jetty.version>
-        <guice.version>4.2.2</guice.version>
-        <slf4j.version>1.7.26</slf4j.version>
-        <jackson.version>2.10.2</jackson.version>
+        <hk2-bridge.version>2.6.1</hk2-bridge.version>
+        <jersey.version>2.36</jersey.version>
+        <jetty.version>10.0.11</jetty.version>
+        <guice.version>5.1.0</guice.version>
```

Comment:
Guice did not need to be upgraded as all tests passed without upgrading it. So it could be downgraded if older version would be preferred.